src/Styleguide/Utils/HorizontalPadding.tsx

```diff
@@ -0,0 +1,26 @@
+import { space, SpacingUnit } from "@artsy/palette"
+import styled from "styled-components"
+import { media } from "../../Components/Helpers"
+
+interface PaddingProps {
+  sizeXS?: SpacingUnit | 0
+  size?: SpacingUnit | 0
+}
+
+const responsivePadding = media.sm<PaddingProps>`
+  padding-right: ${p => space(p.sizeXS as SpacingUnit) || 0}px;
```

Comment:
Should palette be updated to handle `0` correctly? Right now, `0` is not part of the `SpacingUnit` enum, and `space(0)` is `undefined`.

Reply:
If `space(0) == 0` then why not just use `0`?